### PR TITLE
Updated READMe and citations file

### PR DESCRIPTION
**Pull request by ArzaEshita**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,37 +21,33 @@ The updated version of the project [v2.0.0](https://github.com/NCSU-Group7-SE202
...
-Now we support a lot of new file extensions:
-1. Word Document
-2. PDF
-3. Powerpoint
-4. Text
-5. Markdown
+## Phase 1 Features
 
-Upload your lecture pdf, word doc or powerpoint and LectureAid will extract the text, process it, and search the internet for key topics from that lecture. Once it finds relevant results, LectureAid will show a list of questions relevant to your topic, and website links that should answer said questions, and also a wordcloud that highlights key words in the lecture. 
+Upload your lecture pdf, word doc or powerpoint to our user based terminal menu, and LectureAid will extract the text, process it, and search the internet for key topics from that lecture. Once it finds relevant results, LectureAid opens up a browser window with a list of questions relevant to your topic, and website links that should answer said questions, and also a wordcloud that highlights key words in the lecture.
```
Comment (rahulkalita8) [inline README.md:28]: This is wrong no?

File: README.md [hunk 2]
```diff
@@ -21,37 +21,33 @@ The updated version of the project [v2.0.0](https://github.com/NCSU-Group7-SE202
 
 After a long class, ever had to come back home and google everything you supposedly learnt from the lecture handout for that day's class? Ever spend ~30 - 45 minutes just to search through google and compile a list of websites that explain what you need? And then a month later when midterms are around a corner, ever spend that same 30-45 minutes trying to find those websites again cause you forgot to bookmark them? 
 
-Project LectureAid hopes to solve that hassle for you. 
+Project LectureAid hopes to solve that hassle for you.
 
-Now we support a lot of new file extensions:
```
Comment (rahulkalita8) [inline README.md:26]: Why did you delete this?

**Review by rahulkalita8 — CHANGES_REQUESTED** [review]
Please check comments

File: README.md (hunk 2 above)
Comment (ArzaEshita) [reply README.md:26]: Cause it is mentioned in phase 2

File: README.md (hunk 1 above)
Comment (ArzaEshita) [reply README.md:28]: This is what is done is phase 1. Heading is mentioned. Phase is added below

**Review by rahulkalita8 — APPROVED** [review]
